file_formats/fip.md

```diff
@@ -20,46 +20,72 @@ For example:
 
 ```plaintext
 📦 fib
-┣ FIP_DataG_2024-06-05T08_25_33.csv
-┣ FIP_DataIso_2024-06-05T08_25_33.csv
-┣ FIP_DataR_2024-06-05T08_25_33.csv
-┣ FIP_RawG_2024-06-05T08_25_33.bin
-┣ FIP_RawIso_2024-06-05T08_25_33.bin
-┣ FIP_RawR_2024-06-05T08_25_33.bin
-┣ FIP_ROIsG-Iso_2024-06-05T08_25_33.csv
-┗ FIP_ROIsR_2024-06-05T08_25_33.csv
+┣📂 SoftwareEvents
+┃ ┣📜RepositoryStatus.json
+┃ ┗📜Region.json
+┣ green.csv
+┣ red.csv
+┣ iso.csv
+┣ green.bin
+┣ red.bin
+┣ iso.bin
```

Comment:
We are missing the <color>.meta files.
We can omit their documentation since the current standard already "hard-codes" how to open a flat binary file. However, if we want to be safe and be able to resize raw data in the future, it may be important to highlight how these metadata files describe how to open said data: (Width, Height, format, and number of channels)